src/IconButton/IconButton.js

```diff
@@ -164,30 +191,49 @@ class IconButton extends Component {
 
   handleMouseEnter = (event) => {
     this.showTooltip();
-    if (this.props.onMouseEnter) this.props.onMouseEnter(event);
+
+    // Cancel hover styles for touch devices
+    if (!this.state.touch) {
+      this.setState({hovered: true});
+    }
+    if (this.props.onMouseEnter) {
+      this.props.onMouseEnter(event);
+    }
   };
 
-  handleKeyboardFocus = (event, keyboardFocused) => {
-    if (keyboardFocused && !this.props.disabled) {
+  handleTouchStart = (event) => {
+    this.setState({touch: true});
+    this.props.onTouchStart(event);
```

Comment:
This is introduction an error when `onTouchStart` is undefined.

Reply:
Got that too